extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -698,4 +766,29 @@ export class Bind {
 
     return false;
   }
+
+  /**
```

Comment:
Not really happy about this. I'm open to suggestions on how to improve this method.